homeassistant/components/sensor/wunderground.py

```diff
@@ -29,50 +29,562 @@
 
 DEFAULT_LANG = 'EN'
 
-MIN_TIME_BETWEEN_UPDATES_ALERTS = timedelta(minutes=15)
-MIN_TIME_BETWEEN_UPDATES_OBSERVATION = timedelta(minutes=5)
+MIN_TIME_BETWEEN_UPDATES = timedelta(minutes=5)
+
+
+# Helper classes for declaring sensor configurations
+
+class WUSensorConfig(object):
```

Comment:
Hi @Teagan42, thanks for checking the code !!!
The reasons why I decided to have separate classes for storing the configuration is that the WU API is a mess and I didn't wanted to clutter the WUndergroundSensor code by all the different parsing methods for different sensor types. The second reason to have the parsing/configuration for the sensors in separate class is to allow the possibility for having multiple weather stations in the future, where multiple sensors for the same thing (with the very same configuration/parsing) will coexist in parallel, and instantiating the Entity subclasses directly in the SENSOR_TYPES list will make this much more difficult.